internal/bot/bot.go

```diff
@@ -52,6 +52,10 @@ func (b *Bot) Start(ctx context.Context) {
 	defer b.flushLogs()
 	b.botLog.Info("Bot started. Press CTRL + C to quit.")
 
+	if err := b.updateVolumeTraded(ctx); err != nil {
+		panic(fmt.Sprintf("failed to load initial volume traded: %s", err))
+	}
```

Comment:
Perhaps skip this step when the `min_quote_volume_traded` option is not defined or `0`?